build.gradle

```diff
@@ -54,6 +54,9 @@ ext {
 dependencies {
     implementation("org.springframework.boot:spring-boot-starter-web:2.5.5")
     implementation("org.springframework.boot:spring-boot-starter-actuator:$springBootVersion")
+    implementation ("org.springframework.boot:spring-boot-starter-validation:$springBootVersion")
```

Comment:
these validations may not produce error messages in our custom format. Please be careful with that